packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -115,6 +153,57 @@ input:-webkit-autofill {
   line-height: 14px;
 }
 
+.NotifiAlertBox__container {
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.NotifiAlertBox__btn--left,
+.NotifiAlertBox__btn--right {
+  width: 24px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  cursor: pointer;
+  margin-bottom: 0 !important;
+}
+
+.NotifiAlertBox__btn--spacer {
+  width: 24px;
+  height: 24px;
+  padding: 10px;
+  visibility: hidden;
+  margin-bottom: 0 !important;
+}
+
+.NotifiAlertBox__content {
+  flex: 1;
+  display: flex;
+  margin-left: 8px;
+  margin-right: 8px;
+  color: var(--notifi-secondary-font-color);
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 18px;
+  margin-bottom: 0 !important;
```

Comment:
would it better if we remove the !last-child : mb-10? haha

Reply:
actually probably. I'm going to take a pass at it. I think instead of every decendant selector it should be immediate child selector